spec/factories/collection_template.rb

```diff
@@ -24,808 +24,14 @@
-            'DatumName' => 'Datum 2',
-            'DistanceUnits' => 'Fathoms',
-            'Resolutions' => [12.0, 22.0, 32.0]
-          }
-        }
-      },
-      'LocationKeywords' => [{
-        'Category' => 'GEOGRAPHIC REGION',
-        'Type' => 'ARCTIC'
-      }, {
-        'Category' => 'OCEAN',
-        'Type' => 'ATLANTIC OCEAN'
-      }],
-      'TemporalExtents' => [{
-        'PrecisionOfSeconds' => 1,
-        'EndsAtPresentFlag'  => false,
-        'SingleDateTimes'    => ['2015-07-01T00:00:00Z', '2015-12-25T00:00:00Z']
-      }, {
-        'PrecisionOfSeconds' => 10,
-        'EndsAtPresentFlag'  => false,
-        'RangeDateTimes' => [{
-          'BeginningDateTime' => '2014-07-01T00:00:00Z',
-          'EndingDateTime'    => '2014-08-01T00:00:00Z'
-        }, {
-          'BeginningDateTime' => '2015-07-01T00:00:00Z',
-          'EndingDateTime'    => '2015-08-01T00:00:00Z'
-        }]
-      }, {
-        'PrecisionOfSeconds' => 30,
-        'EndsAtPresentFlag'  => false,
-        'PeriodicDateTimes' => [{
-          'Name'                     => 'test 1 Periodic Extent',
-          'StartDate'                => '2015-07-01T00:00:00Z',
-          'EndDate'                  => '2015-08-01T00:00:00Z',
-          'DurationUnit'             => 'DAY',
-          'DurationValue'            => 5,
-          'PeriodCycleDurationUnit'  => 'DAY',
-          'PeriodCycleDurationValue' => 1
-        }, {
-          'Name'                     => 'test 2 Periodic Extent',
-          'StartDate'                => '2016-07-01T00:00:00Z',
-          'EndDate'                  => '2016-08-01T00:00:00Z',
-          'DurationUnit'             => 'MONTH',
-          'DurationValue'            => 4,
-          'PeriodCycleDurationUnit'  => 'MONTH',
-          'PeriodCycleDurationValue' => 2
-        }]
-      }],
-      'TemporalKeywords' => ['Monthly Climatology', 'Weekly Climatology'],
-      'PaleoTemporalCoverages' => [{
-        'StartDate' => '50 Ga',
-        'EndDate'   => '25 Ga',
-        'ChronostratigraphicUnits' => [{
-          'Eon'                    => 'test 1 Eon',
-          'Era'                    => 'test 1 Era',
-          'Epoch'                  => 'test 1 Epoch',
-          'Stage'                  => 'test 1 Stage',
-          'DetailedClassification' => 'test 1 Detailed Classification',
-          'Period'                 => 'test 1 Period'
-        }, {
-          'Eon'                    => 'test 2 Eon',
-          'Era'                    => 'test 2 Era',
-          'Epoch'                  => 'test 2 Epoch',
-          'Stage'                  => 'test 2 Stage',
-          'DetailedClassification' => 'test 2 Detailed Classification',
-          'Period'                 => 'test 2 Period'
-        }, {
-          'Eon' => 'test 3 Eon text 1'
-        }]
-      }],
-      'ISOTopicCategories' => %w[farming climatologyMeteorologyAtmosphere health]
-    }}
+    draft {
+      collection_one.merge(
```

Comment:
I am debating if we should try and make the corresponding change in collection draft factories, or write a tech debt ticket to consolidate later, as the proposals probably could do this also.